src/rust/ensogl/lib/core/src/system/gpu/data/attribute.rs

```diff
@@ -121,11 +121,14 @@ impl {
         })
     }
 
-    /// Disposes instance for reuse in the future. Please note that the disposed data still
+    /// Dispose instance for reuse in the future. Please note that the disposed data still
     /// exists in the buffer and will be used when rendering. It is yours responsibility to hide
-    /// id, fo example by degenerating vertices.
+    /// it, for example by degenerating the mesh (putting all vertices in the same place).
```

Comment:
This docs should be adjusted: it not quite user responsibility to clear data, as this function set them to the default.